frontend/dispatcher.lua

```diff
@@ -210,6 +214,10 @@ local dispatcher_menu_order = {
     "calibre_browse_tags",
     "calibre_browse_series",
 
+    "show_crashlog",
+    "show_taillog",
+    "truncate_crashlog",
```

Comment:
> Maybe we can create a new category "developer" instead of plugin them in device.

I'm fine with them being in Device (no need for another Developer category that most people don't care about - unless we end up adding more?).
Even push them at the end tail of Device, below everything else (below terminal, lasteditedfile, favorites....)

Reply:
And "show_crashlog_tail" rather than "show_taillog".

Reply:
>  unless we end up adding more?

That's the point. IMHO it is better a category than most people ignore than a few options that clutter the category most people use.

Also, some of them can be added from https://github.com/koreader/koreader/pull/6608#issuecomment-683762231

Reply:
May be they don't need to be all available to dispatcher/gestures. show_crashlog_tail might be the most interesting candidate. Full crash log and truncate might have very little use via gestures.
As for the others I might be using, it's so rare I need them that just having them hidden in a far away menu is fine enough :)